### ISSUE TITLE
After date selection from a DatePicker in a Custom Filter Menu the filter is also closed

### ISSUE THREAD

**Issue by joqkey** [telerik/kendo-angular#1481]
## Current behavior
After date selection from a DatePicker in a Custom Filter Menu the filter is also closed.

## Expected behavior
After date selection from a DatePicker in a Custom Filter Menu the filter should stay open.

## Minimal reproduction of the problem with instructions
I am providing a plunker with the described problem.
https://plnkr.co/edit/8qRZ5aOSYwPpWZgb9w6N?p=preview

**Comment by danielkaradachki** [comment]
Indeed, clicks outside the filter menu popup will cause the popup to close and the datepicker popup is rendered in the root component by default. It is rather hard to tell in a generic way whether a clicked element is associated with an element inside the filter menu so we have exposed a way to prevent the filter menu popup from closing. An example demonstrating the scenario with the datepicker is available in the dev version of the site [here](https://www.telerik.com/kendo-angular-ui-develop/components/grid/filtering/reusable-filter/#toc-filter-menu-elements-with-popup).
